src/selectors/index.ts

```diff
@@ -67,3 +81,46 @@ export const selectSchemaFieldDefs = createSelector(
     });
   }
 );
+
+const selectResult = (state: State) => state.present.result.main;
+
+const selectMainSpec = createSelector(
```

Comment:
Why do we need a separate selector here? 

We can simply use

```
{
  data: data,
  transform: getTransforms(filters),
  ...getTopSpecQueryItem(result.modelGroup).spec
}
```

in `selectMainResult`